source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -850,19 +850,16 @@ are parsed from the ismaster response in the obvious way.
 Server Description Equality
 ```````````````````````````
 
-For the purposes of updating topology description and publishing SDAM events,
-two server descriptions having the same address
-MUST be considered equal if and only if the values of
-`ServerDescription`_ fields marked (=) are respectively equal.
+For the purpose of determining whether to publish SDAM events, two server
+descriptions having the same address MUST be considered equal if and only if
+the values of `ServerDescription`_ fields marked (=) are respectively equal.
+Two topology descriptions MUST be considered equal if and only if they have
+the same set of server addresses, and the server descriptions for each address
+are respectively equal as just defined.
 
 This specification does not prescribe how to compare server descriptions
 with different addresses for equality.
 
-Note: Server description for each server MUST be updated (replaced)
```

Comment:
Do we also have to remove this sentence below:

> If the new server description is equal to the previous server description as defined in Server Description Equality, stop the processing.  

It seems like it's saying the same thing as this removed paragraph is.

Reply:
Removed this also.